roles/confluent.common/tasks/redhat.yml

```diff
@@ -36,3 +36,10 @@
     name: "{{redhat_java_package_name}}"
     state: present
   when: install_java|bool
+
+- name: Install unzip
+  yum:
+    name: "{{item}}"
+    state: present
+  loop:
```

Comment:
loop is unnecessary

Reply:
Was just trying to match how we are doing it for the others (debian.yml and ubuntu.yml). Not sure why the others are installing more extras than red hat though.

Reply:
saves two lines to just be 
```
- name: Install unzip
  yum:
    name: unzip
    state: present
```